terraform/variables.tf

```diff
@@ -1,8 +1,24 @@
+# ------------------------------------------------------------------------------
+# Optional parameters
+#
+# These parameters have reasonable defaults.
+# ------------------------------------------------------------------------------
+
 variable "aws_region" {
   description = "The AWS region to deploy into (e.g. us-east-1)."
   default     = "us-east-1"
 }
 
+variable "bucket_name" {
+  description = "The name of the bucket that the user should be allowed to read from."
+  default     = "ncats-3rd-party-packages"
+}
+
+variable "nessus_package_pattern" {
+  description = "The pattern that matches the name of Nessus package objects in the S3 bucket specified by the bucket_name variable."
+  default     = "Nessus-?.?.?-debian6_amd64.deb"
```

Comment:
Some of the Nessus versions will include versions with two characters for a minor version (such as the current version "8.10.0").

Reply:
Good point.  Looking [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-arn-format.html) at the limited jazz available to us, I think `Nessus-*-debian6_amd64.deb` or `Nessus-*.*.*-debian6_amd64.deb` might be the best we can do here.

Reply:
Yup, good thinking @mcdonnnj - updated in https://github.com/cisagov/ansible-role-nessus/pull/8/commits/1dc842c9ee375564cbffb812b3c49ba49314d0e9.